include/nuttx/wireless/gs2200m.h

```diff
@@ -48,6 +48,7 @@
 #include <nuttx/config.h>
 #include <nuttx/irq.h>
 #include <nuttx/wireless/ioctl.h>
+#include <nuttx/wireless/wireless.h>
 
```

Comment:
Do we need to include wireless.h in gs2200m.h?
I think it should be included in gs2200m.c instead, because definitions or types in wireless.h are not used in gs2200m.h

Reply:
Yes, I think it needed. Definitions of wireless.h is needed in nuttx/drivers/wireless/gs2200m.c and apps/wireless/gs2200m/gs2200m_main.c , so I add this include in include/nuttx/wireless/gs2200,h .
I plan to create new pull request to incubator-nuttx-apps correspond to this commit.

Reply:
In that case, please include wireless.h in both gs2200m.c and gs2200m_main.c and remove the above line in gs2200m.h

Reply:
I got it.